spec/sample_data/group_set.json

```diff
@@ -90,6 +90,11 @@
                 "rel": "self",
                 "type": "application/atom+xml",
                 "href": "https://www.google.com/m8/feeds/groups/example%40gmail.com/full/6"
+            },
+            {
+                "rel": "edit",
+                "type": "application/atom+xml",
+                "href": "https://www.google.com/m8/feeds/groups/example%40gmail.com/full/6"
```

Comment:
I also snuck this in here to check if my test passes on a group with an edit link